### DISCUSSION TITLE
6670 bekommt GCC 9.4

### DISCUSSION THREAD

**Discussion by MasterRoCcO** [Freetz-NG/freetz-ng#1022] [Allgemein]
Nur so zur Information, die 6670 hat mit der neuen Labor und Inhaus Version GCC 9.4 im Gepäck

**Comment by fda77** [comment]
Danke für die Info.

Die Patches die AVM nutzt sind im "osp" Paket und kommen dann (für 9.4) nach https://github.com/Freetz-NG/freetz-ng/tree/master/make/toolchain/target/gcc/patches/9.3 

Dazu sollte man noch wissen dass der kernel-gcc aktuell nur für Kernel-Module genutzt wird, da es für aktuelle Geräte in Freetz kein replace-kernel gibt (bis es jemand macht)

**Comment by fda77** [comment]
-> a9b6443a9

Erinner mich an die avm patches wenn es mal sourcen geben sollte!
